package.json

```diff
@@ -86,10 +86,10 @@
 		"tailwindcss": "^3.4.17",
 		"tailwindcss-animate": "^1.0.7",
 		"typescript": "^5.8.3",
-		"vite": "^6.4.1",
+		"vite": "^7.2.2",
 		"vite-plugin-bundle-prefetch": "^0.0.4",
```

Comment:
Maybe this plugin can be dropped entirely. It seems not maintained, and also doesnt give us a benefit rn